### PR TITLE
Create lab10.md

### PR DESCRIPTION
**Pull request by ConnieZhou5**

### CODE REVIEW

File: labs/lab10.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+### Setup
+Fork and clone the Quiz App repository from: [https://github.com/CMU-17-214/f24-lab10](https://github.com/CMU-17-214/f24-lab10) and run
+
+```
+npm install
+npm start
+```
+![Local Image](https://github.com/CMU-17-214/s24-rec09/blob/main/src/image/starterPic.png)
+
```
Comment (jlacomis) [inline labs/lab10.md:22]: This file needs to be added and updated

**Review by jlacomis — CHANGES_REQUESTED** [review]
Just add the one image and I'll merge

**Comment by ConnieZhou5** [comment]
Updated with image path.

**Comment by jlacomis** [comment]
> Updated with image path.
Don't forget to also add the image there.

**Pull request merged**